v3/webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = (env, argv) => {
       client: {
         overlay: {
           errors: true,
-          warnings: false,
+          warnings: true,
```

Comment:
Personally, I don't think this change should be checked in, but we can discuss what the default should be.

Reply:
Yeah, it was an accident. I'll revert the change and hope it comes out in the squash.